src/NUnitTestAdapter/NUnitTestAdapter.cs

```diff
@@ -177,7 +179,15 @@ private TestPackage CreateTestPackage(string assemblyName)
 
             // Always run one assembly at a time in process in it's own domain
             package.Settings[PackageSettings.ProcessModel] = "InProcess";
-            package.Settings[PackageSettings.DomainUsage] = "Single";
+
+            if (Settings.DomainUsage != null)
+            {
+                package.Settings[PackageSettings.DomainUsage] = Settings.DomainUsage;
+            }
+            else
+            {
+                package.Settings[PackageSettings.DomainUsage] = "Single";
```

Comment:
The normal default for NUnit is "Multiple". @CharliePoole should that be updated here, or was it made "Single" for a reason?